tests/Model/PHPClass.php

```diff
@@ -24,6 +24,9 @@ class PHPClass extends BasePHPClass
     /** @var PHPProperty[] */
     public $properties = [];
 
+    /** @since 8.2 */
```

Comment:
`since` can be removed here because this class is a part of model and `$isReadonly` here is just a property of an object and isn't related with php version